test/integration/inserting_test.exs

```diff
@@ -66,16 +104,29 @@ defmodule Oban.Integration.InsertingTest do
     changesets_2 = Enum.map(3..4, &Worker.new(%{ref: &1}))
     changesets_3 = Enum.map(5..6, &Worker.new(%{ref: &1}))
 
-    multi = Multi.new()
-    multi = Oban.insert_all(name, multi, "jobs-1", changesets_1)
-    multi = Oban.insert_all(name, multi, "jobs-2", changesets_2)
-    multi = Multi.run(multi, "build-jobs-3", fn _, _ -> {:ok, changesets_3} end)
-    multi = Oban.insert_all(name, multi, "jobs-3", & &1["build-jobs-3"])
-    {:ok, %{"jobs-1" => jobs_1, "jobs-2" => jobs_2, "jobs-3" => jobs_3}} = Repo.transaction(multi)
+    multi_1 = Multi.new()
+    multi_1 = Oban.insert_all(name, multi_1, "jobs-1", changesets_1)
+    multi_1 = Oban.insert_all(name, multi_1, "jobs-2", changesets_2)
+    multi_1 = Multi.run(multi_1, "build-jobs-3", fn _, _ -> {:ok, changesets_3} end)
+    multi_1 = Oban.insert_all(name, multi_1, "jobs-3", & &1["build-jobs-3"])
+
+    {:ok, %{"jobs-1" => jobs_1, "jobs-2" => jobs_2, "jobs-3" => jobs_3}} =
+      Repo.transaction(multi_1)
 
     assert [%Job{}, %Job{}] = jobs_1
     assert [%Job{}, %Job{}] = jobs_2
     assert [%Job{}, %Job{}] = jobs_3
+
+    start_supervised_oban!(name: Oban, queues: false)
```

Comment:
Will you break this out into a separate test? Also, it could cause `async` issues if we start the default supervisor in another test case. I'll need to keep an eye out for that.

Reply:
Done: https://github.com/sorentwo/oban/pull/755/commits/8794ca0d47eebadd47b655af79fad611e41af337

> Also, it could cause async issues if we start the default supervisor in another test case. I'll need to keep an eye out for that.

Yes, sometimes in local:
```
1) test name is by default `Oban` (ObanTest)
     test/oban_test.exs:9
     match (=) failed
     code:  assert {:ok, pid} = start_supervised({Oban, repo: Oban.Test.Repo})
     left:  {:ok, pid}
     right: {:error, {{:already_started, #PID<0.751.0>}, {:child, :undefined, Oban, {Oban, :start_link, [[repo: Oban.Test.Repo]]}, :permanent, false, :infinity, :supervisor, [Oban]}}}
     stacktrace:
       test/oban_test.exs:10: (test)
```

Reply:
@oo6 Did you make a change? If you're busy, I'll gladly make the fixes and a new test myself.